inst/include/Rcpp/sugar/functions/median.h

```diff
@@ -0,0 +1,297 @@
+// -*- mode: C++; c-indent-level: 4; c-basic-offset: 4; tab-width: 8 -*-
```

Comment:
I have a much newer version of this line but then I never said anything official about it...